frontend/components/cards/BookCard.tsx

```diff
@@ -1,30 +1,54 @@
+'use client'
```

Comment:
Is this intended?

Reply:
Nawp. It's a [convention](https://beta.nextjs.org/docs/rendering/server-and-client-components#convention) for the NextJSv13 app directory to use client side rendering. 😄

Reply:
Oooh. Haven't upgraded yet. But good to know!